src/location.js

```diff
@@ -10,18 +11,22 @@ const parseJsonApiUrl = (fromUrl) => {
         path: url.pathname,
         query: {
             filter: query.get('filter'),
-            include: query.get('include'),
-            fields: query.get('fields'),
-            sort: query.get('sort'),
+            include: query.get('include')||[],
+            fields: query.get('fields')||{},
+            sort: query.get('sort')||[],
```

Comment:
I had added something like this in the branch I was working on, otherwise the default value is `null` which makes it difficult to assume any methods are possible (like `map`)